src/main/java/frc/robot/commands/teleopcommands/ClimberUp.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
This whole file is indented with two spaces but every other file uses four? Please fix the formatting here